### DISCUSSION TITLE
course/new-theory-of-the-body/up-hierarchy

### DISCUSSION THREAD

**Discussion by giscus** [thepopupschool/thepopupschool#5] [Courses]
# course/new-theory-of-the-body/up-hierarchy/

... putting the mind back into the body and the body back into nature

https://thepopupschool.org/course/new-theory-of-the-body/up-hierarchy/

**Comment by bonnittaroy** [comment]
Thanks so much for getting this site up and running. Looking forward to discussions. Had trouble solving the car problem when trying to verify my account. Didn't find it intuitive, so had to try a couple of times... haha.

**Reply by namjul** [reply]
Yea these puzzles are weird. Apart maybe form the sign-up experience this form of an embedded discussion board feels good. With discord there is no easy way to embed it into the site, so we would have links to the respective module channels.
